crates/core/c8y_api/src/utils.rs

```diff
@@ -2,7 +2,9 @@ pub mod bridge {
 
     use mqtt_channel::Message;
 
-    pub const C8Y_BRIDGE_HEALTH_TOPIC: &str = "tedge/health/mosquitto-c8y-bridge";
+    // FIXME: doesn't account for custom topic root, use MQTT scheme API here
```

Comment:
Could you create an issue with the list of areas where custom topics are not supported? Others can also contribute to that list and once it's complete, we can address them all at once.

Reply:
Made the issue:
- #2357